package.json

```diff
@@ -9,6 +9,8 @@
     "url": "git://github.com/woocommerce/woocommerce-gateway-stripe.git"
   },
   "devDependencies": {
+    "@types/react": "^17.0.18",
+    "@types/react-dom": "^17.0.9",
```

Comment:
I see a mismatch between the version used for the types and the version used for the actual packages